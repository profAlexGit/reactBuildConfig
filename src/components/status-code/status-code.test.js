import React from 'react';
import { configure } from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow';
import Adapter from 'enzyme-adapter-react-16';

import StatusCode from './status-code.jsx';
configure({ adapter: new Adapter() });

const props = {
  code: 301,
  children: <div />
};

describe('<StatusCode />', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    const tree = renderer.render(<StatusCode {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
