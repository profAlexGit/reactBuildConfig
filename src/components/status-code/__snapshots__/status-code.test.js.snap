// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<StatusCode /> renders correctly 1`] = `
<Route
  render={[Function]}
/>
`;
